spotify/album.json

```diff
@@ -9,7 +9,15 @@
             "tokenServerUrl": "https://accounts.spotify.com/api/token"
         },
         "params": {
-            "q": true
+            "q": true,
+            "limit": false,
+            "offset": false
+        },
+        "pagination":{
+            "type":"limit",
+            "anchor_key":"offset",
```

Comment:
As we discussed, please use universal parameters for offset and limit like https://github.com/sfu-db/dataprep/issues/138.

i.e.`_limit` and `offset` for all the APIs.